### PR TITLE
Landing Page Cards 3rd PR

### PR DESCRIPTION
**Pull request by AmanGandhi2305**

### CODE REVIEW

File: src/shared/components/cards-button/cards-button.component.ts [hunk 1]
```diff
@@ -1,10 +1,66 @@
...
+      }
+      else{
+        return;
+      }
+    }
+
+  }
+
+  hoverChanges(){
+    return {
```
Comment (Trusha-Tatvasoft) [inline src/shared/components/cards-button/cards-button.component.ts]: what's use of this method?

File: src/shared/components/icons/icons.component.ts [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 @Component({
   selector: 'app-icons',
   templateUrl: './icons.component.html',
   styleUrls: ['./icons.component.scss']
 })
-export class IconsComponent {
+export class IconsComponent implements OnInit {
+  @Input() iconName !:string;
+  @Input() iconValue !:string;
+  ifClassApply = true ;
```
Comment (Trusha-Tatvasoft) [inline src/shared/components/icons/icons.component.ts]: give meaning Full name

File: src/shared/components/icons/icons.component.html [hunk 3]
```diff
@@ -1 +1,4 @@
-<p>icons works!</p>
+<div [ngClass]="ifClassApply ? 'iconDiv':''">
+  <mat-icon>{{iconName}}</mat-icon>
+  <span *ngIf="ifClassApply" class="iconValueTag">{{iconValue}}</span>
+</div>
```
Comment (Trusha-Tatvasoft) [inline src/shared/components/icons/icons.component.html:4]: iconValue  you can directly check here iconValue != ""

File: src/shared/components/mission-avg-rating/mission-avg-rating.component.ts [hunk 4]
```diff
@@ -1,10 +1,21 @@
...
+  public stars: boolean[] = [];
 
+  constructor(){}
+
+  ngOnInit(){
+    this.AvgRatingFill();
+  }
+
+  AvgRatingFill(){
+    this.stars = Array(5).fill(false).map((_, index) => index < Math.floor(this.rating));
```
Comment (Trusha-Tatvasoft) [inline src/shared/components/mission-avg-rating/mission-avg-rating.component.ts:19]: method and and var name should be in camel case

**Review by Trusha-Tatvasoft — APPROVED** [review]

File: src/shared/components/cards-button/cards-button.component.ts (hunk 1 above)
Comment (AmanGandhi2305) [reply src/shared/components/cards-button/cards-button.component.ts]: sorry madam, I forgot to remove it.
Right now there is no usable

**Review by Trusha-Tatvasoft — APPROVED** [review]

**Review by ShubhamPadiyaTatvasoft — APPROVED** [review]

**Pull request merged**